packages/supabase_flutter/lib/src/local_storage_web.dart

```diff
@@ -1,19 +1,17 @@
 // ignore: avoid_web_libraries_in_flutter
 import 'dart:html' as html;
 
-import 'package:supabase_flutter/supabase_flutter.dart';
-
 final _localStorage = html.window.localStorage;
 
-Future<bool> hasAccessToken() async =>
-    _localStorage.containsKey(SharedPreferencesLocalStorage.persistSessionKey);
+Future<bool> hasAccessToken(String persistSessionKey) async =>
+    _localStorage.containsKey(persistSessionKey);
 
-Future<String?> accessToken() async =>
-    _localStorage[SharedPreferencesLocalStorage.persistSessionKey];
+Future<String?> accessToken(String persistSessionKey) async =>
+    _localStorage[persistSessionKey];
 
-Future<void> removePersistedSession() async =>
-    _localStorage.remove(SharedPreferencesLocalStorage.persistSessionKey);
+Future<void> removePersistedSession(String persistSessionKey) async =>
+    _localStorage.remove(persistSessionKey);
 
-Future<void> persistSession(persistSessionString) async =>
-    _localStorage[SharedPreferencesLocalStorage.persistSessionKey] =
-        persistSessionString;
+Future<void> persistSession(
```

Comment:
On JS client, the session is stored as an object rather than a String. Could we mimic the behavior here for Flutter web?

https://github.com/supabase/gotrue-js/blob/master/src/GoTrueClient.ts#L1765

Reply:
Ah, didn't realize that. The chrome debug tool made it look like it was an object, so I just assumed it was stored as an object! Great catch!

Reply:
I tried it locally with an Next.js web app and Flutter web app, and the session is persisted 🎉 Thanks for the amazing work!